app/controllers/miq_action_controller.rb

```diff
@@ -0,0 +1,608 @@
+class MiqActionController < ApplicationController
+  before_action :check_privileges
+  before_action :get_session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  include Mixins::GenericSessionMixin
+  include Mixins::BreadcrumbsMixin
+  include Mixins::PolicyMixin
```

Comment:
Okay, I get it now, but are you using the method defined in the previous change in this file?

Reply:
I just don't want to have a mixin that holds methods that are not actually shared or used multiple times.

Reply:
This mixin has methods that are being called from multiple places.